test/cmd/admin.sh

```diff
@@ -148,6 +151,22 @@ oadm policy reconcile-cluster-role-bindings --confirm
 # Ensure --additive-only=false removes customized users from the binding
 oadm policy reconcile-cluster-role-bindings --additive-only=false --confirm
 [ ! "$(oc get clusterrolebindings/basic-users -o json | grep custom-user)" ]
+echo "admin-reconcile-cluster-role-bindings: ok"
+
+os::cmd::expect_success "oc create -f test/extended/fixtures/roles/policy-roles.yaml"
+os::cmd::expect_success "oc get rolebinding/basic-users"
+os::cmd::expect_success "oc delete role/basic-user"
+os::cmd::expect_failure "oc get rolebinding/basic-users)"
+os::cmd::expect_success "oc create -f test/extended/fixtures/roles/policy-clusterroles.yaml"
+os::cmd::expect_success "oc get clusterrolebinding/basic-users2"
+os::cmd::expect_success "oc delete clusterrole/basic-user2"
+os::cmd::expect_failure "oc get clusterrolebinding/basic-users2)"
+os::cmd::expect_success "oc policy add-role-to-user edit foo"
+os::cmd::expect_success "oc get rolebinding/edit"
+os::cmd::expect_success "oc delete clusterrole/edit"
+os::cmd::expect_failure "oc get rolebinding/edit"
+os::cmd::expect_success "oadm policy reconcile-cluster-roles --confirm"
```

Comment:
@stevekuznetsov are you rebased on latest master (re: https://github.com/openshift/origin/commit/2fb8bad59c64eed19e5bfffbfd5df3fb8dadab3f#commitcomment-14629095)?